docs/application/native/guides/applications/overview.md

```diff
@@ -44,6 +44,10 @@ When your application becomes visible again, the `app_resume_cb()` callback is i
 - All applications on top of your application in the window stack finish.
 - An alarm is triggered for your application, bringing it to the front and hiding other applications.
 
+> **Note**
+>
+> Since Tizen 5.5, you can get the display state by using the app_get_display_state() function when the app_resume_cb() callback or the app_pause_cb() callback is invoked.
+
```

Comment:
Reni-90 left a comment:

#### **[Reni-90](/Reni-90) ** [ 15 days ago ](#r255827975) Member

Rephrase as:
From Tizen 5.5, you can get the display state using `app_get_display_state()` when the `app_resume_cb()` callback or the `app_pause_cb()` callback is invoked.

I left a comment:
#### **[journey2w](/journey2w) ** [ 9 days ago ](#r257537704) Member
@Reni-90
"From Tizen 0.0, ..." is correct but we usually use "Since Tizen 00.00, ...". You should've referred the guides of Tizen other modules.
It would be good not to change this phrase, if there is no grammatical error.

---
I think you change the sentence as follow:
```
Since Tizen 5.5, you can get the display state using `app_get_display_state()` when the `app_resume_cb()` callback or the `app_pause_cb()` callback is invoked.
```

Reply:
done

Reply:
@journey2w 
https://github.com/Samsung/tizen-docs/pull/556#discussion_r258824227

If you feel so. Then, we will go as per your suggestion.
Thank you for the explanation.